sentry_sdk/integrations/asgi.py

```diff
@@ -145,6 +134,23 @@ def __init__(
         else:
             self.__call__ = self._run_asgi2
 
+    def _capture_exception(self, exc):
+        # type: (Exception) -> None
+        event, hint = event_from_exception(
+            exc,
+            client_options=sentry_sdk.get_client().options,
+            mechanism={"type": self.mechanism_type, "handled": False},
+        )
+        sentry_sdk.capture_event(event, hint=hint)
+
+    def _capture_lifespan_exception(self, exc):
+        # type: (Exception) -> None
+        return self._capture_exception(exc)
+
+    def _capture_request_exception(self, exc):
```

Comment:
I am a bit unsure of the difference between these two functions. Please add docstrings to both to clarify the distinction